### PR TITLE
Issue2 main approximation

### PR DESCRIPTION
**Pull request by JorgeHBatista**
Although it has all the points from the Definition of Done from Issue 2, it isn't responsive enough to be marked as completed

Captura de pantalla de 2022-12-12 11-22-26

### CODE REVIEW

**Comment by carlosper88** [comment]
Hi @JorgeHBatista! I'll try to review your PRs every now and then (when I have some time)

First thing I'd like to see in a frontend PR, specially if it involves the addition of new screens / modifications in the UI is for it to include some screenshots of the work done.

Apart from that, I see that you're using .css files. That's completely fine and it's necessary to understand how css works. Nevertheless, modern sites are usually built using some third party frameworks like tailwinds or chakra-ui. These frameworks help you customize your components very easily, without the need of using super big css files. It would be interesting that you take a look at them and try to include one of them to the project. Over time, you'll see how the development process speeds up, and the code gets more maintainable, as normally css rules override one another making a single change a mess as the codebase grows

One thing I specially like from the chakra-ui, for example, is this:
https://chakra-ui.com/docs/styled-system/customize-theme

With that you can declare a "variant" for a button for example, and every time you want to use that same style, you just do:
`<Button variant="base" />`
(note that it's a Button imported from chakra-ui, not the html one)

This way we get ride of all of the css stuff and code gets super clean

File: src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { StartingPage } from './pages/StartingPage';
+import { LogInRegister } from './pages/LogInRegister';
+import { RegisterLegal } from './pages/RegisterLegal';
+import { RegisterName } from './pages/RegisterName';
+import { RegisterBirthDate } from './pages/RegisterBirthDate';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+
+const root = ReactDOM.createRoot(document.getElementById('root')!);
+root.render(
+  <BrowserRouter>
```
Comment (carlosper88) [inline src/index.tsx:11]: 👏

File: src/pages/LogInRegister.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import '../assets/css/LogInRegister.css';
+
+//Components imports
```
Comment (carlosper88) [inline src/pages/LogInRegister.tsx:3]: No need for you to add these type of comments, 100% of imports go at the beginning of the file, so no need to add a comment 😉

File: src/components/Logo.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+interface Props {
+  color: string
+};
+
+export const Logo = ({color}: Props) => {
+  return (
+    <img 
+      className = 'logo'
+      src = {color === 'blue' ? blueLogo : whiteLogo}
```
[the remark this answers is not in the document]
Comment (JorgeHBatista) [reply src/components/Logo.tsx:14]: Sure, it's a better idea which I didn't take into count when working on it, but I can fix it in a moment

File: src/pages/LogInRegister.tsx (hunk 2 above)
Comment (JorgeHBatista) [reply src/pages/LogInRegister.tsx:3]: I see, thank you so much for your feedback! I uploaded the different images into the different PR. Now I will fix every point I replied to you in this PR. Which is for you the best way to continue, to keep working on the same branch to keep it all in the same PR or to create another one and have different PR?

Thank you so much, as I am not 100% clear which is the best option I'll take a look to the other comments before to continue working on this PR

Comment (carlosper88) [reply src/pages/LogInRegister.tsx:3]: Hi @JorgeHBatista!
Sorry for the late reply. I've been on vacation last week. You can do the fixes in this branch, the we merge after approval, and then you can keep the other PRs to merge to main, as this code will be already in main.
Take into account that with the latest changes in main, you'll have some conflicts to fix in your branch

**Pull request closed without merging**